snap/snapcraft.yaml

```diff
@@ -0,0 +1,34 @@
+name: wsstat
+adopt-info: wsstat
+license: MIT
+summary: wsstat is a websocket traffic monitoring tool.
```

Comment:
Perhaps "endpoint monitoring" would make more sense here? One does not really "monitor traffic" by manually sending one-off requests to endpoints.